### PR TITLE
Some fixes

### PR DESCRIPTION
**Pull request by atravert**
few fixes mostly related to errors in the documentation

- colab: the set_colab.sh does not work anymore, but as pip install works, scp can be installed easily, spectrochempy_data also (usning git_clone). These updated procedure are described in install_colab.rst. The video is alos outdated.  
- mcrals: bug fix
- analysis/peak_finding 
- example plot with opus
- importIR

### CODE REVIEW

**Comment by atravert** [comment]
Note that there are also few errors on https://www.spectrochempy.fr/latest/userguide/processing/fourier2d.html

**Comment by fernandezc** [comment]
> Note that there are also few errors on https://www.spectrochempy.fr/latest/userguide/processing/fourier2d.html
> 
> which I did not look at :-)

I would love to have time to fix many thing related to NMR :)

File: docs/userguide/analysis/peak_finding.py [hunk 1]
```diff
@@ -468,7 +468,7 @@
     table_property = "  ".join(
         [f"{properties[key][i].m:>10.3f}" for i in range(len(peaks))]
     )
-    title = f"{key:>.16} ({properties[key][0].u:>.6K})"
```
Comment (fernandezc) [inline docs/userguide/analysis/peak_finding.py:471]:
```python 
title = f"{key:>.16} ({properties[key][0].u:~P})"
```  
will be nicer 🙂

**Review by fernandezc — APPROVED** [review]
👍

File: docs/userguide/analysis/peak_finding.py (hunk 1 above)
Comment (fernandezc) [reply docs/userguide/analysis/peak_finding.py:471]: Todo also for the first line

Comment (fernandezc) [reply docs/userguide/analysis/peak_finding.py:471]: print(f'{"peak_position (cm⁻¹)":>26}: {table_pos}')

I copied the unicode directly

**Comment by fernandezc** [comment]
> > Note that there are also few errors on https://www.spectrochempy.fr/latest/userguide/processing/fourier2d.html
> > which I did not look at :-)
> 
> I would love to have time to fix many thing related to NMR :)

But I just tried and indeed this need to be fixed!!!  I look at this soon.

**Pull request merged**